File: Modul 4/Soal 4/PRAK304-2210817120012-Noorlaini Marratain.c
#include <stdio.h>

int main()
{
    int c;
    scanf("%d", &c);
    if(c>=1 && c<=9){
        printf("Satuan\n");}
    else if(c>=10 && c<=19){
        printf("Belasan\n");}
    else if(c>=20 && c<=99){
        printf("Puluhan\n");}
    else if (c<1){
        printf("nol\n");}
    else {
        printf("Anda Menginput Melebihi Limit Bilangan\n");}
}

File: Modul 4/Soal 4/PRAK304-2210817120012-Noorlaini Marratain.py
c=int(input())
if c>=1 and c<=9:
    print("Satuan\n")
elif(c>=11 and c<=19):
    print("Belasan\n")
elif(c>=20 and c<=99):
    print("Puluhan\n")
elif(c<=0):
    print("nol\n")
else:
    print("Anda Menginput Melebihi Limit Bilangan\n")